Source/WebKit/UIProcess/API/Swift/WebPage.swift

```diff
@@ -101,12 +103,14 @@ public class WebPage_v0 {
 
     @ObservationIgnored
     lazy var backingWebView: WKWebView = {
-        let webView = WKWebView(frame: .zero)
+        let webView = WKWebView(frame: .zero, configuration: WKWebViewConfiguration(wrapping: configuration))
```

Comment:
Wrapping would also be nice because it'd mean that `public let configuration: Configuration` could then be inverted to return like `{ Configuration(storage: backingWebView.configuration) }` which just helps a little bit with canonicalization.